config/former.php

```diff
@@ -0,0 +1,184 @@
+<?php return array(
+
+	// Markup
+	////////////////////////////////////////////////////////////////////
+
+	// Whether labels should be automatically computed from name
+	'automatic_label'         => true,
```

Comment:
This is not a good practice. Better be explicit for field labels, that can be extended with i18n in future to translate the site in bengali.